configs/clip/metafile.yml

```diff
@@ -0,0 +1,272 @@
+Collections:
+  - Name: CLIP
+    Metadata:
+      Architecture:
+        - Attention Dropout
+        - Convolution
+        - Dense Connections
+        - Dropout
+        - GELU
+        - Layer Normalization
+        - Multi-Head Attention
+        - Scaled Dot-Product Attention
+        - Tanh Activation
+    Paper:
+      URL: https://arxiv.org/abs/2201.09792
+      Title: Learning Transferable Visual Models From Natural Language Supervision
+    README: configs/clip/README.md
+
+Models:
+  - Name: vit_base_patch16_clip_384.openai_ft_in12k_in1k
```

Comment:
Please rename the model name according to https://mmclassification.readthedocs.io/en/dev-1.x/advanced_guides/convention.html
For example:
`clip-vit-base-p16_openai-pre_3rdparty_in1k`
`clip-vit-base-p16_laion2b-in12k-pre_3rdparty_in1k-384px`